sw/host/spiflash/ftdi_spi_interface.cc

```diff
@@ -106,22 +109,82 @@ bool FtdiSpiInterface::TransmitFrame(const uint8_t *tx, uint8_t *rx,
     std::cerr << "Unable to start spi transaction." << std::endl;
     return false;
   }
+
   uint8_t *tmp_rx = ::Transfer(spi_->ctx, tx_local.data(), size);
+  // We're not using the result of this read, so free it right away.
+  if (tmp_rx) {
```

Comment:
I can't remember. Is this actually a new buffer created by `::Transfer`?

Reply:
The definition is in the vendored in MPSSE code:

https://github.com/lowRISC/opentitan/blob/acc57b715809e4cccf0c712e47d73e0ad8734f21/sw/host/vendor/mpsse/mpsse.c#L865-L907

You can see it's malloced in that function.

Having a `free` call in C++ code does seem a bit nasty but I can't see a clean way around it (other than patching mpsse to include an `mpsse_buf_free` or similar but that would just directly call `free` anyway)